lightkurve/utils.py

```diff
@@ -293,3 +300,72 @@ def plot_image(image, ax=None, scale='linear', origin='lower',
     if show_colorbar:
         plt.colorbar(cax, ax=ax, norm=norm, label=clabel)
     return ax
+
+
+
+def query_catalog(coordinate, catalog="KIC", radius=0.5):
+        """
+        Returns an astropy table of the sources inside a radius query.
+        Current catalogs supported are KIC, EPIC and Gaia DR2.
+
+        Parameters
+        -----------
+        coordinate : astropy.coordinates.SkyCoord
+            Coordinate to query around.
+        catalog: string
+            Indicate catalog assigned for mission. If Kepler, catalog will be KIC
+            if K2 catalog is EPIC.
+        radius: float
+            Radius of cone search centered on the target in arcminutes.
+            Default radius is 0.5 arcmin.
+
+        Returns
+        -------
+        result : astropy.table
+            Astropy table with the following columns
+        ID : astropy.table.column (KIC & EPIC)
+            Catalog ID from catalog.
+        RAJ200: astropy.table.column (KIC & EPIC)
+            Right ascension [degrees]
+        DEJ2000: astropy.table.column (KIC & EPIC)
+            Declination [deg]
+        pmRA: astropy.table.column (KIC & EPIC)
+            Proper motion for right ascension [mas/year]
+        pmDEC: astropy.table.column (KIC & EPIC)
+        Kpmag: astropy.table.column (KIC & EPIC)
+            Magnitude in Kepler band [mag]
```

Comment:
Indent the column definitions to fit inside the indent block for `result`.

Reply:
@AndyTza , same as above!